apps/web/src/utils/metatags.ts

```diff
@@ -0,0 +1,24 @@
+import { deserializePage } from "./page-helper";
+
+type Page = {
+  page: ReturnType<typeof deserializePage>;
+};
+
+//TODO: add dynamic images
+
+export const formatMetaTags = ({ page }: Page) => {
+  const defaultMetaTags = {
+    description:
+      "setup your web3 profile and connect with others by turning your social tokens into social signals.",
+    image: "https://images.unsplash.com/photo-156",
+  };
+
+  const prefixedTitle = page.title ? `denoted/${page.title}` : "denoted";
```

Comment:
You think of this as a breadcrumb?

Reply:
Yeah, inspiration from @ericzakariasson. But a good idea for previews!